### ISSUE TITLE
hass.io addon support

### ISSUE THREAD

**Issue by Jolanrensen** [d8ahazard/Phlex#199]
Hass.io is a popular home automation operating system that utilizes docker and allows for addons to be created: https://home-assistant.io/developers/hassio/addon_tutorial/
As I have seen, Phlex supports docker as well and I tried to create an addon using the Dockerfile given. However it seems to not work out of the box. Firstly the LABEL command fails, when I comment that out the COPY command fails as well and finally RUN returns 1. So yeah... Any ideas?

**Comment by bezmi** [comment]
I've tried to integrate Phlex into a docker container with nginx and it works, but I'm having an issue where resetting the docker container a fresh log on screen and resets the Phlex API token even though the Phlex files are stored in persistent storage. This also happens when I run without a docker container just with nginx on Arch Linux. In contrast, when running under XAMPP, the server "remembers" me after a restart and Phlex stays logged in. I also opened an [issue](https://github.com/d8ahazard/docker-phlex/issues/15) over at docker-phlex.
